napi.h

```diff
@@ -269,8 +269,9 @@ namespace Napi {
 
     napi_valuetype Type() const; ///< Gets the type of the value.
 
-    bool IsUndefined() const;   ///< Tests if a value is an undefined JavaScript value.
     bool IsNull() const;        ///< Tests if a value is a null JavaScript value.
+    bool IsUndefined() const;   ///< Tests if a value is an undefined JavaScript value.
+    bool IsNullOrUndefined() const; ///< Tests if a value is a null or undefined JavaScript value.
```

Comment:
Maybe `IsNullish` would be more concise. With the definition of `??` from [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) as (emphasis mine)

> The nullish coalescing operator (??) is a logical operator that returns its right-hand side operand when its left-hand side operand **is null or undefined**, and otherwise returns its left-hand side operand.